tests/test_common.py

```diff
@@ -122,3 +121,33 @@ def test_units(self, prod, lats, lons, wind_speed_units):
         assert ret['header']['wind_speed_units'] == wind_speed_units
 
         assert ws_kph == round(ws_other*multiplier)
+
+    @pytest.mark.parametrize("prod", [
+        DeepCyc(config=ClientConfig(config_section="reask-demo")),
+        Metryc(config=ClientConfig(config_section="reask-demo")),
+    ])
+    @pytest.mark.parametrize("lats,lons,status", [
+        ([30, 30], [-90, -90.1], {'FORBIDDEN'}),
+        ([-20.5, 18.5], [148.5, 122.2], {'OK', 'FORBIDDEN'}),
+        ([18.4, 18.5], [122.2, 122.2], {'OK'}),
+    ])
+    def test_tcwind_perms(self, prod, lats, lons, status):
+
+        ret = prod.tcwind_events(lats, lons)
+        df = gpd.GeoDataFrame.from_features(ret)
+        assert set(df.status) == status
+
+    @pytest.mark.parametrize("prod", [
+        DeepCyc(config=ClientConfig(config_section="reask-demo")),
```

Comment:
demo user's section will come from `DEMO_USERNAME` so it should follow instead of hardcoding 'reask-demo' section.